project2/styles.css

```diff
@@ -0,0 +1,119 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: rgb(255, 255, 255);
+    color: rgb(0, 0, 0);
+    font-family: 'Be Vietnam', sans-serif;
+}
+
+section {
+    display: inline-block;
+    margin: auto;
+    padding-bottom: 15px;
+    width: 400px;
+
+}
+
+h2,
+ul,
+p {
+    padding-left: 5px;
+    padding-right: 5px;
+}
+
+a:link,
+a {
+    color: black;
```

Comment:
On line 7 you set the `color` of the `body` to be:
`    color: rgb(0, 0, 0);`

Note that this is the _same_ as
`    color: black;`

In general, you want to avoid referring to the same color in two ways because it makes it harder to maintain -- the only reason go use RGB here would be if you needed the transparency of RGBA.